patches/path.patch

```diff
@@ -0,0 +1,37 @@
+--- ./github.com/jipanyang/gnxi/utils/xpath/path.go
```

Comment:
>---  [](http://example.com/codeflow?start=0&length=4)

Could you add some file level comment to explain why a patch is needed, and how do you achieve it?
ref: [https://stackoverflow.com/a/20911434/2514803](https://stackoverflow.com/a/20911434/2514803)